src/components/network/network-table.js

```diff
@@ -241,7 +263,13 @@ const NetworkTable = (props) => {
         );
     }
 
-    function renderSubstationsTable() {
+    const showSelectedColumn = (key) => {
+        return selectedListName.includes(intl.formatMessage({ id: key }))
```

Comment:
don't use formatted messages in comparisions

Reply:
Done with the refactoring